core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -1919,7 +1920,7 @@ private long createFileWithSingleBlock(AlluxioURI uri) throws Exception {
 
   private void startServices() throws Exception {
     mRegistry = new MasterRegistry();
-    mSafeModeManager = new DefaultSafeModeManager();
+    mSafeModeManager = new DefaultSafeModeManager(new ManualClock(), false);
```

Comment:
for tests which don't want to use safe mode, we could use a dummy `TestSafeModeManager` which extends `SafeModeManager` and always returns `false` for `isInSafeMode()`. Then we don't need the extra boolean parameter in the constructor.

Reply:
Done.